### PR TITLE
Add vb sample

### PR DESCRIPTION
**Pull request by Youssef1313**
Added vb samples and fixed code that was contradicting

> Each sentence in the string assigned to the Exception.Message property should end in a period.

### CODE REVIEW

File: docs/standard/exceptions/best-practices-for-exceptions.md [hunk 1]
```diff
@@ -173,6 +193,18 @@ catch (Exception ex)
 }
 ```
 
+```vb
+Catch ex As Exception ex
```
Comment (rpetrusha) [inline docs/standard/exceptions/best-practices-for-exceptions.md]:
```suggestion
Catch ex As Exception
```

File: docs/standard/exceptions/best-practices-for-exceptions.md [hunk 2]
```diff
@@ -173,6 +193,18 @@ catch (Exception ex)
...
+    from.RollbackTransaction(withdrawalTrxID)
+    Throw New TransferFundsException("Withdrawal failed.", innerException:=ex) With
+    {
+        .From = from,
+        .To = to,
```
Comment (rpetrusha) [inline docs/standard/exceptions/best-practices-for-exceptions.md]:
```suggestion
        .[To] = [to],
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you for adding the missing Visual Basic examples, @Youssef1313. I've suggested a number of changes so that the code compiles.

**Comment by Youssef1313** [comment]
I've committed one suggestion. For the others, I didn't check, but I think it should be `[to]` because visual basic has a reserved word `to` ?

**Comment by rpetrusha** [comment]
Yes, to is a reserved word, @Youssef1313. I think that all of my remaining suggestions replace `to` with `[to]`, With that, the code compiles and executes successfully.

**Comment by Youssef1313** [comment]
> Yes, to is a reserved word, @Youssef1313. I think that all of my remaining suggestions replace `to` with `[to]`, With that, the code compiles and executes successfully.

Oh sorry for that, I thought I already had the square brackets and read the suggestions backwards. Going to apply the changes now.

**Comment by rpetrusha** [comment]
Thanks for the additional changes, @Youssef1313. I'll approve and merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**